src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -98,17 +114,62 @@ const TabPanel = (props: TabPanelProps) => {
   );
 };
 
+type LocationState = { numUpdates: number };
+
 const CCPDashboardPage = ({ match }: RouteComponentProps<TParams>) => {
   const classes = useStyles();
+  const location = useLocation<LocationState>();
+
   const { eventId, ccpId, patientId } = match.params;
+  const { numUpdates } = location.state || { numUpdates: 0 };
```

Comment:
Does this account for when location.state exists but it has no `numUpdates` property?